File: frontEnd/src/app/models/xu-li-muon-online-dto.ts
export class XuLiMuonOnlineDTO {
    maHV: string;
    idMuonOnline: number;
    tenSach: string[];
    ngayMuon: Date;
    ngayTra: Date;
    maNV: string;
  
    // Constructor to initialize the class properties
    constructor(
      maHV: string,
      idMuonOnline: number,
      tenSach: string[],
      ngayMuon: Date,
      ngayTra: Date,
      maNV: string
    ) {
      this.maHV = maHV;
      this.idMuonOnline = idMuonOnline;
      this.tenSach = tenSach;
      this.ngayMuon = ngayMuon;
      this.ngayTra = ngayTra;
      this.maNV = maNV;
    }
}

File: BackEnd/BookLand/src/main/java/com/bookland/dto/xuLiMuonOnlineDTO.java
package com.bookland.dto;

import java.util.Date;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class xuLiMuonOnlineDTO {
	private String maHV;
    private int idMuonOnline;
    private List<String> tenSach;
    private Date ngayMuon;
    private Date ngayTra;
    private String maNV;

    
}
